config/config.exs

```diff
@@ -0,0 +1,3 @@
+use Mix.Config
+
+config :tesla, :adapter, Tesla.Adapter.Hackney
```

Comment:
Would it be possible to allow the user of the sdk to set the adapter via an environment variable? Or is there functionality or adapter options that only work with hackney?